Countries/Apollo.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Apollo.swift
+//  Countries
+//
+//  Created by Khrystyna Matasova on 24.11.2021.
+//
+
+import Foundation
+import Apollo
+
+class Apollo {
+    
+    static let shared = Apollo()
+    
+    let client: ApolloClient
+
+    private init() {
+        client = ApolloClient(url: URL(string: "https://countries.trevorblades.com/")!)
```

Comment:
What you will do when API will become unavailable?
I mean FORCE UNWRAP

Reply:
oh no! how could I miss it!?
thank you